docs/spelling_wordlist.txt

```diff
@@ -75,3 +75,4 @@ workspace
 worktree
 worktrees
 zsh
+Perlbrew
```

Comment:
Nitpick, but it would be nice to keep this file in alphabetical order as much as possible.

Reply:
I sorted the contents of the file, using sort on the command line. Don note it was not entirely alphabetical